homeassistant/components/ecobee/services.yaml

```diff
@@ -1,104 +1,211 @@
 create_vacation:
+  name: Create vacation
   description: >-
     Create a vacation on the selected thermostat. Note: start/end date and time must all be specified
     together for these parameters to have an effect. If start/end date and time are not specified, the
     vacation will start immediately and last 14 days (unless deleted earlier).
   fields:
     entity_id:
-      description: ecobee thermostat on which to create the vacation (required).
+      name: Entity
+      description: ecobee thermostat on which to create the vacation.
+      required: true
       example: "climate.kitchen"
+      selector:
+        entity:
+          integration: ecobee
+          domain: climate
     vacation_name:
-      description: Name of the vacation to create; must be unique on the thermostat (required).
+      name: Vacation name
+      description: Name of the vacation to create; must be unique on the thermostat.
+      required: true
       example: "Skiing"
+      selector:
+        text:
     cool_temp:
-      description: Cooling temperature during the vacation (required).
+      name: Cool temperature
+      description: Cooling temperature during the vacation.
+      required: true
       example: 23
+      selector:
+        number:
+          min: 5
+          max: 100
```

Comment:
I believe this is still out of range. 

7 & 35 degrees celsius would be ~ 45 - 95 degrees Fahrenheit